### PR TITLE
[Workloads] Microsoft.Workloads Preview Release

### PR DESCRIPTION
**Pull request by Jing-song**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command

Microsoft.Workloads
aaz model : https://github.com/Azure/aaz/pull/179
close https://github.com/Azure/azure-cli/issues/25550

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Review by mohit1706 — APPROVED** [review]
Looks good, verified command examples -  Approved

**Comment by prajjwalsrivastav** [comment]
Looks good, verified command examples - Approved

**Comment by mohit1706** [comment]
@jsntcy by when it will released and by when the documentation will be available for customers?

File: src/workloads/setup.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup, find_packages
+
+
+# HISTORY.rst entry.
+VERSION = '0.1.0-preview'
```
Comment (jsntcy) [inline src/workloads/setup.py:13]: Will you want it be released as `preview` as I saw the swagger used is GA?

Comment (sagarkeswani) [reply src/workloads/setup.py:13]: Yes swagger is GA but we want to release the CLI client in Preview to get initial customer feedback before removing preview tag.

Comment (kairu-ms) [inline src/workloads/setup.py:13]:
```suggestion
VERSION = '0.1.0a1'
```

**Review by kairu-ms — APPROVED** [review]

File: src/workloads/setup.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 
 # HISTORY.rst entry.
-VERSION = '0.1.0-preview'
```
Comment (jsntcy) [inline src/workloads/setup.py]: Since we already add `a1` as preview label, I think we should use `1.0.0a1` instead as `0.x.x` means preview as well.

**Pull request merged**